### DISCUSSION TITLE
character rendering problem

### DISCUSSION THREAD

**Discussion by gkjain58** [community/community#137309] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

In Adobe typing programs, while I type in Hindi or Sanskrit with Noto Sans Devanagari fonts the r is rendered like a caret symbol, whereas it should be a slant, like wise single quotes and coma appears as sharp slants whereas these are wanted with rounded caps. On the other hand, in MS word these problems are not there. Please suggest if there is any solitions.

**Comment by LiteBrite82** [comment]
Hi @gkjain58 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
